tests/test_errors.py

```diff
@@ -0,0 +1,39 @@
+"""Module for testing the running of the API"""
```

Comment:
This is a test...What does it have to do with the commit?

Reply:
Some of my changes were poorly committed. Thanks for pointing it out.